src/components/dashboard/sidebar/SubsidiesSummary.jsx

```diff
@@ -85,6 +93,30 @@ const SubsidiesSummary = ({
   const hasAvailableSubsidyOrRequests = (
     hasActiveLicenseOrLicenseRequest || hasAssignedCodesOrCodeRequests || learnerCreditSummaryCardData
   );
+  const hasAutoAppliedLearnerCreditPolicies = (
+    redeemableLearnerCreditPolicies?.filter(policy => policy.policyType !== POLICY_TYPES).length > 0
+  );
```

Comment:
Instead of `policy.policyType !== POLICY_TYPES` i think i would be `policy => policy.policyType !== POLICY_TYPES.ASSIGNED_CREDIT`.
I am saying that after looking into the logic that Adam has implemented [here](https://github.com/openedx/frontend-app-learner-portal-enterprise/pull/897/files#diff-9b8f6c24fbf8c0f299a677e6043a72ca1f7d098f5640f5f8b26a66c0047e15baR118)